packages/sit-onyx/src/styles/fonts.scss

```diff
@@ -1,6 +1,9 @@
 .onyx-text {
-  font-weight: 400;
-  font-size: 1rem;
+  font: {
+    weight: 400;
+    size: 1rem;
+    family: var(--onyx-font-family);
```

Comment:
I didn't add the font family intentionally because it breaks the inheritance...you can take a look at the typography page in the VitePress docs which now will no longer work with the monospace font

Reply:
Removed my changes and created follow-up ticket